client/src/pages/automation/project/components/node-details-tabs/OutputTab.tsx

```diff
@@ -36,48 +38,82 @@ const AnimateSpin = ({className}: {className?: string}) => (
+
+    const value = property.name && getNestedObject(sampleOutput, selector);
+
+    return (
+        <div>
+            <div className="group relative inline-flex items-center rounded-md p-1 pr-5 text-sm hover:bg-gray-100">
+                <span title={property.type}>{TYPE_ICONS[property.type as keyof typeof TYPE_ICONS]}</span>
+
+                <span className="px-2">{label}</span>
+
+                {value && typeof value !== 'object' && (
+                    <div className="flex-1 text-xs text-muted-foreground">{value}</div>
+                )}
+
+                <div className="absolute right-0.5">
+                    <ClipboardIcon
+                        aria-hidden="true"
+                        className="invisible size-4 cursor-pointer bg-background text-gray-400 group-hover:visible"
+                        onClick={() => copyToClipboard(`$\{${workflowNodeName}.${selector}}`)}
+                    />
+                </div>
+            </div>
+        </div>
+    );
+};
 
 const SchemaProperties = ({
-    path,
+    parentPath,
     properties,
     sampleOutput,
+    workflowNodeName,
 }: {
+    parentPath?: string;
     properties: Array<PropertyType>;
-    path?: string;
     sampleOutput: object;
+    workflowNodeName: string;
 }) => (
     <ul className="ml-2 h-full">
         {properties.map((property, index) => {
-            const value =
-                property.name &&
-                getNestedObject(sampleOutput, `${path ? path + '.' : ''}${property.name}`.replace('/', '.'));
-
             return (
                 <li className="flex flex-col" key={`${property.name}_${index}`}>
-                    <div className="flex items-center space-x-3">
-                        <PropertyField data={property} label={property.name!} />
-
-                        {value && typeof value !== 'object' && (
-                            <div className="flex-1 text-xs text-muted-foreground">{value}</div>
-                        )}
-                    </div>
+                    <PropertyField
+                        label={property.name!}
+                        parentPath={parentPath}
+                        property={property}
+                        sampleOutput={sampleOutput}
+                        workflowNodeName={workflowNodeName}
+                    />
 
                     {property.properties && !!property.properties.length && (
                         <div
                             className="ml-3 flex flex-col overflow-y-auto border-l border-gray-200 pl-1"
                             key={property.name}
                         >
                             <SchemaProperties
-                                path={`${path ? path + (property.name ? '.' : '') : ''}${property.name || '[index]'}`}
+                                parentPath={`${parentPath ? parentPath + (property.name ? '.' : '') : ''}${property.name || '[index]'}`}
```

Comment:
zbunjujuća linija koda, valjda izgleda bolje kad nije u ovom github sučelju, iako se dva puta ponavlja.

ja bi osobno izbacija logiku konkatenacije stringa u varijablu, ali ću ostavit tebi na volju.

var1 = `parentPath + (property.name ? '.' : '')`
var2 = `${property.name || '[index]'}`

tako nešto